charts/castai-evictor/values.yaml

```diff
@@ -61,6 +61,10 @@ fullnameOverride: castai-evictor
 
 rbac:
   enabled: true
+  # When Evictor Helm chart is upgraded from Cluster Controller we don't have
+  # permissions to update ClusterRoleBinding or ClusterRole, so we need a way to skip it.
+  volumes:
+    enabled: true
```

Comment:
[question]: should we enable it by default?

Reply:
Yes, when customer installs the chart during onboarding they should get new version.